barretenberg/cpp/src/barretenberg/vm/avm/generated/flavor.hpp

```diff
@@ -302,11 +304,11 @@ class AvmFlavor {
 
     class AllConstRefValues {
       public:
-        using DataType = const FF&;
```

Comment:
Argh... I can't do this if I want to avoid copying the fields.

I had to do it because, while I DO want `const FF&` members, I want `get_all` etc to return `RefArray<const FF>` and not `RefArray<const FF&>` which is not possible.

I'll have to see how to solve this. Maybe removing the & when passing the type to RefArray (which is a bit of a hack).

Reply:
Done, with `std::remove_reference_t`.